include/clang/AST/Expr.h

```diff
@@ -4514,6 +4514,120 @@ class GenericSelectionExpr : public Expr {
   friend class ASTStmtReader;
 };
 
+/// \brief Represents a Checked C bounds expression.
+class BoundsExpr : public Expr {
```

Comment:
Okay; that's probably a good candidate for mentioning along with the other high-level implementation design decisions that you were planning to write up.  I assume other folks will ask the same question in the future, and it might be helpful to anticipate that :)

---
In reply to: [69053660](https://github.com/Microsoft/checkedc-clang/pull/8#discussion_r69053660) [](ancestors = 69053660)